docs/book/src/user/quick-start.md

```diff
@@ -161,10 +161,9 @@ before getting started with Cluster API. See below for the expected settings for
 {{#tabs name:"tab-installation-infrastructure" tabs:"AWS,Azure,DigitalOcean,Docker,GCP,vSphere,OpenStack,Metal3,Packet"}}
 {{#tab AWS}}
 
-Download the latest binary of `clusterawsadm` from the [AWS provider releases] and make sure to place it in your path. You need at least version v0.5.5 for these instructions.
-Instructions for older versions of clusterawsadm are available in [Github][legacy-clusterawsadm].
+Download the latest binary of `clusterawsadm` from the [AWS provider releases] and make sure to place it in your path.
 
-The clusterawsadm command line utility assists with identity and access management (IAM) for Cluster API Provider AWS.
+The [clusterawsadm][clusterawsadm] command line utility assists with identity and access management (IAM) for [Cluster API Provider AWS][capa].
```

Comment:
Looks odd that one of these has the single `[blah]` where the others have `[blah][blah]` with the references below? Is the single `[blah]` acceptable/preferred? Should we make them consistent?

Reply:
You are right. i'll fix